tasks/skylab/StarAlign.wdl

```diff
@@ -337,31 +338,52 @@ task STARsoloFastq {
       --soloUMIdedup 1MM_Directional_UMItools \
       --outSAMtype BAM SortedByCoordinate \
       --outSAMattributes UB UR UY CR CB CY NH GX GN \
-      --soloBarcodeReadLength 0
```

Comment:
Do we typically run star twice? Is it possible to run it just once?

Reply:
We run it twice for the count_exons = true. With the latest version of STAR, we should be able to run both parameters in a single STAR run. We haven't yet played with that functionality.